docs/_docs/includes.md

```diff
@@ -97,15 +103,18 @@ This include contains 5 parameters:
 You could also use `if` tags to safeguard scenarios where either the parameter is optional or where the author doesn't include the parameter:
 
 ```liquid
-{% raw %}<figure>{% if {{include.url}} %}<a href="{{include.url}}">
-{% endif %}<img src="{{include.file}}" {% if {{include.max-width}}
-%} style="max-width: {{include.max-width}}" {% endif %}
-alt="{{include.alt}}" />{% if {{include.url}} %}</a>{% endif %}
-{% if {{include.caption}} %}<figcaption>{{include.caption}}
-</figcaption>{% endif %}</figure>{% endraw %}
+{% raw %}<figure>
+   {% if {{ include.url }} %}<a href="{{ include.url }}">{% endif %}
+   <img src="{{ include.file }}" {% if {{ include.max-width }} %}
+   style="max-width: {{ include.max-width }}" {% endif %}
+   alt="{{ include.alt }}" />{% if {{ include.url }} %}</a>{% endif %}
+   {% if {{ include.caption }} %}
+   <figcaption>{{ include.caption }}</figcaption>
+   {% endif %}
+</figure>{% endraw %}
```

Comment:
ahh, now we're coming into something interesting territory. I tested the code locally and have been using includes like this for months in my doc projects. But I didn't have spaces around the variables. It turns out this spacing matters when the variable appears inside an if statement.

This works:

```
<figure>
   {% if {{include.url}} %}<a href="{{include.url}}">{% endif %}
   <img src="{{include.file}}" {% if {{include.max-width}} %}
   style="max-width: {{include.max-width }}" {% endif %}
   alt="{{include.alt}}" />{% if {{include.url}} %}</a>{% endif %}
   {% if {{include.caption}} %}
   <figcaption>{{include.caption}}</figcaption>
   {% endif %}
</figure>
```

This doesn't:

```
<figure>
   {% if {{ include.url }} %}<a href="{{ include.url }}">{% endif %}
   <img src="{{ include.file }}" {% if {{ include.max-width }} %}
   style="max-width: {{ include.max-width }}" {% endif %}
   alt="{{ include.alt }}" />{% if {{ include.url }} %}</a>{% endif %}
   {% if {{ include.caption }} %}
   <figcaption>{{ include.caption }}</figcaption>
   {% endif %}
</figure>
```

The difference is with the spacing in the page variables. 

I was under the impression that `{% if {{include.url}} %}` and `{% if {{ include.url }} %}` would be rendered the same, since having the include outside of an `{% if ... %}` doesn't impose the same requirements about spacing. Outside of if statements, `{{ include.file }}` and `{{include.file}}` work equally well. 

I can remove the spacing for variables inside of `if` statements in the code and add a little explanatory note. Is this a bug in Jekyll, or a feature?